account-keys/src/account_keys.rs

```diff
@@ -403,6 +403,11 @@ impl AccountKey {
         self.subaddress_spend_private(DEFAULT_SUBADDRESS_INDEX)
     }
 
+    /// The private spend key for the change subaddress.
+    pub fn change_subaddress_spend_private(&self) -> RistrettoPrivate {
```

Comment:
Using "change" as the first word in this method name makes it seem like it's meant to be changing the subaddress. I realize that your following the convention set by the `subaddress_spend_private` for the DEFAULT_SUBADDRESS_INDEX, but I think it might be clearer if you changed the name to `get_change_subaddress_spend_private`. That way it's clear that this method is a getter.

Reply:
I named it this way to be consistent with default_subaddress_spend_private. Should I update that too for consistency? Or do you think I should leave it as is?

Reply:
I would prefer you update it.

Reply:
Thanks!